### PR TITLE
Make remove_column and remove_index idem_potent

### PR DESCRIPTION
**Pull request by coisnepe**

### CODE REVIEW

File: lib/safe-pg-migrations/plugins/statement_retrier.rb [hunk 1]
```diff
@@ -13,6 +13,10 @@ module StatementRetrier
       end
     end
 
+    def remove_column(table_name, column_name, type = nil, options = {})
+      retry_if_lock_timeout { super }
+    end
```
Comment (matthieuprat) [inline lib/safe-pg-migrations/plugins/statement_retrier.rb]: Why this change?

Comment (coisnepe) [reply lib/safe-pg-migrations/plugins/statement_retrier.rb]: It otherwise threw this error:

```
  2) Error:
SafePgMigrationsTest#test_change_table:
StandardError: An error has occurred, all later migrations canceled:

wrong number of arguments (given 3, expected 2)
    safe-pg-migrations/lib/safe-pg-migrations/plugins/idem_potent_statements.rb:15:in `remove_column'
    safe-pg-migrations/lib/safe-pg-migrations/plugins/statement_retrier.rb:16:in `block (3 levels) in <module:StatementRetrier>'
```

Comment (matthieuprat) [reply lib/safe-pg-migrations/plugins/statement_retrier.rb]: Do you have this error on `master`?

Comment (matthieuprat) [reply lib/safe-pg-migrations/plugins/statement_retrier.rb]: @coisnepe I ran tests on your branch without this change and they are all green 🍏. Are you sure this change is needed?

(Also, we should now have Travis statuses if you push again.)
